src/Register.php

```diff
@@ -67,8 +67,9 @@ public static function register_widgets() {
 	private static function register_widget( $widget_class ) {
 		register_widget( $widget_class );
 
-		if ( method_exists( $widget_class, 'post_registration' ) ) {
-			call_user_func( [ $widget_class, 'post_registration' ] );
+		$callback = [ $widget_class, 'post_registration' ];
+		if ( is_callable( $callback, true ) ) {
```

Comment:
I had to change an instance of this to method_exists because is_callable didn't work as expected (it failed when the method didn't exist at at all). Have you tested it in all cases - eg the method does exist, the method doesn't exist and the method exists but is private?

I also read that method_exists is faster than is_callable, though I'm not too concerned about performance in this case.